test/conftest.py

```diff
@@ -20,25 +20,49 @@
 
 def pytest_addoption(parser: pytest.Parser) -> None:
     parser.addoption("--full-test-mode", action="store_true",
-                     help="Enable full test mode")
+                     help="Run all tests")
+    parser.addoption("--default-test-mode", action="store_true",
+                     help="Run all tests except the very slow ones")
     parser.addoption("--fast-test-mode", action="store_true",
-                     help="Enable fast test mode")
+                     help="Run only tests without LLM inference")
+    parser.addoption(
+        "--llm-test-only",
+        action="store_true",
+        help="Run only tests with LLM inference except the very slow ones",
+    )
+    parser.addoption("--very-slow-test-only", action="store_true",
+                     help="Run only the very slow tests")
 
```

Comment:
Now the pytest action run a test for three times. Add two options "--llm-test-only" and "--very-slow-test-only" to allow finer control.